extensions/binding/org.eclipse.smarthome.binding.onewire/ESH-INF/thing/temperature.xml

```diff
@@ -7,7 +7,6 @@
 		<label>Temperature sensor</label>
 		<description>A 1-wire temperature sensor (DS18x20)</description>
 		<channels>
-			<channel id="temperature" typeId="temperature" />
```

Comment:
Can't we drop the while `channels` element if it does not contain any child node?